cmd/rook/rook/rook.go

```diff
@@ -181,3 +182,35 @@ func GetOperatorBaseImageCephVersion(context *clusterd.Context) (string, error)
 
 	return output, nil
 }
+
+// GetInternalOrExternalClient will get a Kubernetes client interface from the KUBECONFIG variable
+// if it is set, or from the operator pod environment otherwise.
+func GetInternalOrExternalClient() kubernetes.Interface {
+	kubeconfig := os.Getenv("KUBECONFIG")
+
+	var (
+		config *rest.Config
+		err    error
+	)
+
+	if kubeconfig != "" {
+		for _, kConf := range strings.Split(kubeconfig, ":") {
+			config, err = clientcmd.BuildConfigFromFlags("", kConf)
+			if err != nil {
```

Comment:
Is this intended if there is not an error?
```suggestion
			if err == nil {
```

Reply:
Oh, good catch! Running on autopilot 😂